### DISCUSSION TITLE
能将pdf导出成word吗?

### DISCUSSION THREAD

**Discussion by cooltnt** [wmjordan/PDFPatcher#89] [交流]
如题! 前几天还在找这方面工具, 找了一圈, 没有好用的免费工具

**Comment by wmjordan** [comment]
这个太难了，我搞不定

**Comment by wmjordan** [comment]
微软Office已自带PDF转换为DOC的功能

**Reply by cooltnt** [reply]
嗯, 我试了下, 转换后里面的文字, 是图片形式, 和我那天找的一个python开源处理结果相似

**Reply by cooltnt** [reply]
office2013 有的pdf转word, 文字能编辑
